examples/with-cookie-auth/pages/api/profile.js

```diff
@@ -18,11 +19,12 @@ export default async (req, res) => {
       const data = Object.assign({}, { avatarUrl: js.avatar_url }, js)
       return res.status(200).json({ data })
     } else {
-      return res.status(response.status).send(response.statusText)
+      // https://github.com/developit/unfetch#caveats
+      const error = new Error(response.statusText)
+      error.response = response
+      throw error
     }
-  } catch ({ statusText, statusCode }) {
-    const err = new Error(statusText)
-    err.statusCode = statusCode
-    return err
+  } catch ({ response }) {
+    return res.status(response.status).json({ message: response.statusText })
```

Comment:
Add a check here to make sure that response exists 🙏

Reply:
Note that you're doing the same in other places, it's always recommended to handle the normal error just in case it didn't throw where you wanted it to throw, but this one is the most important one to change 👍

Reply:
I added some changes for this